# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/diligent-engine/include-dirs.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2352252..c906454 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,6 +2,9 @@ cmake_minimum_required (VERSION 3.6)
 
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
+include_directories(SYSTEM /usr/X11R7/include/)
+include_directories(/usr/include/nlohmann)
+
 # Generate XCode schema files
 set(CMAKE_XCODE_GENERATE_SCHEME TRUE)
 # Make malloc write 0xAA to newly allocated memory and 0x55 to deallocated memory
